src/components/Process/CSP/basics.tsx

```diff
@@ -44,8 +46,17 @@ export const useTwoFactorAuth = <T extends number>(process: PublishedElection, s
         body: JSON.stringify(payload),
       })
       if (!response.ok) {
-        throw new Error((await response.json()).error)
+        const errorMessage = (await response.json()).error
+
+        const translatedMessage = /participant not found in the census/i.test(errorMessage)
+          ? t('csp.errors.participantNotFound', {
```

Comment:
does not respect the naming, should be

`csp.errors.participant_not_found`, though I'd say to remove the `errors` if there aren't anymore.